x-pack/plugin/watcher/src/main/java/org/elasticsearch/xpack/watcher/Watcher.java

```diff
@@ -855,7 +855,7 @@ public String getFeatureDescription() {
         return "Manages Watch definitions and state";
     }
 
-    private Settings getWatchesIndexSettings() {
+    private static Settings getWatchesIndexSettings() {
```

Comment:
Nit: "build" or "create" instead of "get". Same for other private static methods in this class.